### ISSUE TITLE
Misspelled in /data...

### ISSUE THREAD

**Issue by tux1c** [tux-mind/tf201-dev#2]
Hi and sorry for bothering you,

I feel like a complete retard, I've recently tried to install GNU/Linux on my TF201, and used your tools.
But I have misspelled mmcblk in /data/.kernel.d/android and now I can't access Android (nor I can Gentoo, but that's a seperate problem).

I have access to the recovery, I also backed up all of my nvflash data.
Is there anyway I can bypass kernel_chooser \ uninstall it \ mount the TF201 on my PC (Windows and Linux)?
Thanks in advance!

**Comment by chrischdi** [comment]
You could go to recovery and use adb to fix the /data/.kernel.d/android :-)
Do a "pull /data/.kernel.d/android"
Fix the file and do "push android /data/.kernel.d/android"

**Comment by tux1c** [comment]
@chrischdi god, I love you!
But now, I have another problem ><
kernel_chooser tries to search /mnt/boot/android_kernel even though I specified it to use simply /boot/android
I also tried to mkdir /mnt/boot and upload android_kernel and initrd there, but as soon as the tablet reboots /mnt vanishes. 

Any clue?

**Comment by tux-mind** [comment]
@Tux1c `/` is a tmpfs where the bootloader extract the initramfs/initrd, so all your changes get lost after reboot.

can you post your kernel_chooser configuration file?

sorry if i cannot help you so much but i don't work on this project from more then an year.

**Comment by tux1c** [comment]
@tux-mind hi, yes I am aware that the project is pretty much dead, and I thank you for helping me.
I am using the default config file that's suggested in the wiki:
Android
/dev/mmcblk0p8:/boot/android_kernel:/boot/android_initrd

I should get home soon and try to set the root manually to /sdcard.
I will edit this post if you don't reply by the time I get home :)

Edit: okay, I'm starting to figure things out.
it appears to mount /dev/mmcblk0p8 directly on /mnt, I think.
now, I know that mmcblk0p8 is /sdcard, I moved the files there but it still tells me they don't exist.
I tried testing with another file (somefile.txt that was in /sdcard) and it said it doesn't find it, too.
I guess it simply fails to mount /dev/mmcblk0p8?
